locales/en/index.yml

```diff
@@ -628,6 +628,12 @@ wallet:
     bankName: Name of the bank
     noName: Name unavailable
     loading: Loading bank list...
+  addBancomat:
+    title: "Add bancomat {{current}} of {{length}}"
+    screenTitle: "Do you want to add this card?"
+    bodySingular: "Abbiamo trovato 1 carta a te intestata. Procedi con l'aggiunta di questa carta cliccando Aggiungi o clicca Salta per annullare l'operazione."
```

Comment:
It should be in english

Reply:
@Undermaken Yep, I noticed while developing the following story to connect the bancomat screen with redux, I'm going to fix this here too